platform/centec-arm64/linux-kernel-arm64.mk

```diff
@@ -0,0 +1,5 @@
+# linux kernel package for centec arm64
+
+LINUX_KERNEL_DTB = linux-dtb_1.0.0-1_arm64.deb
+$(LINUX_KERNEL_DTB)_URL = https://github.com/CentecNetworks/sonic-binaries/raw/master/$(PLATFORM_ARCH)/kernel/$(LINUX_KERNEL_DTB)
```

Comment:
is there any reason we are still using a binary? I thought we can use sonic build kernel?

Reply:
Thanks for the review. We'll look into it and try to compile the dts by the kernel building.